remotespark/kernels/sparkkernel/kernel.js

```diff
@@ -0,0 +1,7 @@
+define(['base/js/namespace'], function(IPython){
+        var onload = function() {
+            IPython.CodeCell.config_defaults.highlight_modes['magic_text/x-sql'] = {'reg':[/^%%sql/]};
```

Comment:
* Reinstalled kernels, opened new notebooks in PySpark and Spark

* Created multiple cells starting with %%sql to verify that syntax highlighting worked in every cell starting with %%sql

* Checked Javascript console to ensure there were no unexpected errors